src/include/lsm.h

```diff
@@ -192,6 +192,13 @@ struct __wt_lsm_manager {
  */
 #define	WT_LSM_AGGRESSIVE_THRESHOLD	2
 
+/*
+ * The minimum size for opening a tree: three chunks, plus one page for each
+ * participant in up to three concurrent merges.
```

Comment:
It took me a bit to parse and understand the comment. It would be good to remove "three" from the comment and make it flexible. I also changed the name to make clear what the number unit is. What do you think of:
```
#define WT_LSM_CONCURRENT_MERGE_MAX 3
#define WT_LSM_CHUNKS_MIN 3
/*
 * The minimum size for opening a tree: The minimum number of chunks, plus one page
 * for each participant for the maximum number of concurrent merges.
 */
#define WT_LSM_TREE_MINIMUM_BYTES(...)
```

Reply:
Thanks -- I'll tuck that change into my next "lint" PR.